### ISSUE TITLE
Permission popup every few hours

### ISSUE THREAD

**Issue by carstenhag** [wulkano/kap#809]
**macOS version:** 10.14.6
**Kap version:** Version 3.0.1

#### Steps to reproduce
Wait some hours. Popup pops up.

#### Current behaviour
Permission popup with "Kap wants permission to... " or similar pops up every few hours, but even after giving it that, it pops up again in some hours.

Pretty sure this was introduced with the 3.0 update, as before I didn't have this issue.

#### Expected behaviour
It only pops up when it has to install an update.

**Comment by skllcrn** [comment]
I see that you are on an older version of macOS @carstenhag, have you experienced the issue in a more recent version of the OS?

**Comment by carstenhag** [comment]
I only have one Mac, and I have not updated yet, I know :p. Not sure when I'll be updating, I'm still waiting.

**Comment by andantonelli** [comment]
Same problem here. And I have macOs regular updated at 10.15.3
Thanks!

**Comment by karaggeorge** [comment]
We are using [this](https://www.electronjs.org/docs/api/system-preferences#systempreferencesaskformediaaccessmediatype-macos) which says if it has asked once it won't ask again and the user has to manually go change it in the Preferences. I'll do some investigation on my system (10.14) and I might need to open an issue with them

**Comment by karaggeorge** [comment]
Just to make sure, what type of permissions dialog are you seeing? Is it the microphone one, or something different? 

@andantonelli @carstenhag

**Comment by andantonelli** [comment]
@karaggeorge this:
<img width="461" alt="Schermata 2020-03-27 alle 14 53 40" src="https://user-images.githubusercontent.com/17927246/77763115-de681a80-703a-11ea-9ebb-bd982d3875bf.png">

Translation: "Kap is trying to add a new helper. Insert password..."

**Comment by carstenhag** [comment]
Exactly. Probably doesn't happen to many people, only to those which have a second user with admin rights.

**Comment by karaggeorge** [comment]
Huh, well that is not what I was expecting. I have never seen that dialog. I'll try creating a new user on my laptop and trying to see if I can replicate

**Comment by karaggeorge** [comment]
Apparently that seems to be an issue for other Apps as well in a similar setup. Can you guys try some of these solutions and see if that fixes it:

- https://apple.stackexchange.com/a/271123
- https://github.com/atom/atom/issues/7058#issuecomment-283698557

**Comment by carstenhag** [comment]
```
sudo chown -R carstenh:staff Kap.app
pwd
   /Applications/Kap.app
ls -la
  total 0
  drwxr-xr-x   3 carstenh  staff    96 Nov 30 21:25 .
  drwxrwxr-x+ 77 root      admin  2464 Mar 26 16:25 ..
  drwxr-xr-x   9 carstenh  staff   288 Nov 30 21:29 Contents
```

**Comment by carstenhag** [comment]
<img width="585" alt="Bildschirmfoto 2020-03-30 um 09 30 38" src="https://user-images.githubusercontent.com/3776997/77886546-59167d00-7269-11ea-86f2-32768b1caed4.png">

Still showing up :(

**Comment by carstenhag** [comment]
Visual Studio Code also shows this popup, but only when it's preparing for an update. After the update was completed, it doesn't show up again. Maybe look into how they have solved this?


```
$ ls -la Kap.app Visual\ Studio\ Code.app
Kap.app:
total 0
drwxr-xr-x   3 carstenh  staff    96 30 Nov 21:25 .
drwxrwxr-x+ 77 root      admin  2464 30 Mär 12:50 ..
drwxr-xr-x   9 carstenh  staff   288 30 Nov 21:29 Contents

Visual Studio Code.app:
total 0
drwxr-xr-x   3 root  wheel    96 23 Mär 18:48 .
drwxrwxr-x+ 77 root  admin  2464 30 Mär 12:50 ..
drwxr-xr-x   9 root  wheel   288 24 Mär 09:09 Contents
```

**Comment by andantonelli** [comment]
> * https://apple.stackexchange.com/a/271123

This seems work for me! About a day without a notice.
Thanks!

**Comment by karaggeorge** [comment]
@carstenhag looks like Kap and VS Code have a different owner and group (crastenh/staff vs root/wheel) any chance you can have Kap under the same owner and try again?

**Comment by carstenhag** [comment]
Before I had ran chown, the owners were root:admin.

**Comment by carstenhag** [comment]
In the meanwhile I have updated to the latest Mac version. The issue does not occur anymore, but I can't pinpoint whether the Mac update fixed it or my chown attempt.

Should we close this or take a look at how other electron apps do it? VS code, Postman and Abstract show the same pop-ups, but they never show the Kap behavior.

**Comment by carstenhag** [comment]
It's still happening, not sure why I had commented back then that it's fixed.

**Comment by carstenhag** [comment]
This bug is still annoying me, so I analyzed it a bit more extensively:

I was on 3.2.0, everytime I started Kap it nagged me to update. The permissions were:

```
drwxr-xr-x   3 root  wheel    96 31 Mär  2020 .
drwxrwxr-x+ 47 root  admin  1504 25 Jan 14:13 ..
drwxr-xr-x   9 root  wheel   288 31 Mär  2020 Contents
```

then I ran

```
sudo chown -R -R carstenh:staff Kap.app
```
and the permissions were fixed:
```
ls -la Kap.app

total 0
drwxr-xr-x   3 carstenh  staff    96 31 Mär  2020 .
drwxrwxr-x+ 47 root      admin  1504 25 Jan 14:13 ..
drwxr-xr-x   9 carstenh  staff   288 31 Mär  2020 Contents
```

I started up Kap, it nagged me, I entered my password and then it updated to 3.2.2.

Then, I checked for the permissions again and it was reset to root:wheel again!

```
total 0
drwxr-xr-x   3 root  wheel    96  9 Sep 18:27 .
drwxrwxr-x+ 47 root  admin  1504 27 Jan 21:43 ..
drwxr-xr-x   9 root  wheel   288  9 Sep 18:32 Contents
```

So it really seems like the permissions are reset to `root` after upgrading successfully (?!). That's probably also the reason why it seemed fixed in April 2020, then broken in December 2020 again.
